ktlint-rules/src/software/aws/toolkits/ktlint/rules/LogLambdaRule.kt

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.ktlint.rules
+
+import com.github.shyiko.ktlint.core.Rule
+import org.jetbrains.kotlin.com.intellij.lang.ASTNode
+import org.jetbrains.kotlin.psi.KtCallExpression
+import org.jetbrains.kotlin.psi.psiUtil.getCallNameExpression
+import org.jetbrains.kotlin.psi.psiUtil.getReceiverExpression
+import org.jetbrains.kotlin.psi.psiUtil.referenceExpression
+
+class LogLambdaRule : Rule("log-not-lazy") {
```

Comment:
I'd definitely call this `LazyLogRule` or something - the word "Lambda" in the CR description totally threw me off!

Reply:
#758